ast/builtins.go

```diff
@@ -1715,6 +1716,19 @@ var JSONIsValid = &Builtin{
 	Categories: encoding,
 }
 
+var RenderTemplate = &Builtin{
+	Name:        "render_template",
```

Comment:
This is currently under the `encoding` section. Is this better suited under `strings`. For ex `strings.render_template`

Reply:
Agreed, I think it does make more sense under strings